rosdep/python.yaml

```diff
@@ -1216,6 +1216,14 @@ python-control-pip:
   ubuntu:
     pip:
       packages: [control]
+python3-cookiecutter:
+  debian:
+    buster: [python3-cookiecutter]
+    stretch: [python3-cookiecutter]
+  fedora: [python3-cookiecutter]
+  ubuntu:
+    '*': [python3-cookiecutter]
+    focal: null
```

Comment:
The last remaining issue that CI is reporting is that this `python3-` prefixed key needs to be moved with the rest of the python3 keys below since the file is maintained in lexicographical sort order.